spec/policies/recipe_policy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require './spec/support/features/clearance_helpers'
+
+RSpec.describe RecipePolicy, type: :request do
+  let(:user) { create(:user) }
+  let(:other_user) { create(:user) }
+  let(:recipe) { create(:recipe, user: user) }
+
+  context 'when unauthenticated' do
+    it 'redirects to login' do
+      get '/recipes'
```

Comment:
The tests you've added to this file don't belong here. They should be in the recipe request spec. This file would be used to test the policy class directly without any http request